app/assets/javascripts/admin/services.js

```diff
@@ -44,4 +44,31 @@ adminServices.factory('Rooms', function($resource) {
         mesg: function() { return data.mesg },
         status: function() { return data.status }
     };
+}).factory("QRLink", function($resource) {
+
+    // Takes room IDs, will generate URLs that correspond to the 'qr' view in directory_objects
```

Comment:
Can this comment be cleaned up? Does it take an ID or IDs? What is a sample of the input and what is the sample output? Do I give it 'R1103' or '1103' or ['1003', '121']? Does it return relative or absolute URLs?